coordinator/README.md

```diff
@@ -11,11 +11,27 @@ npm install
 npm run serve
 ```
 
+### Directories
+- `/models` Typescript interface/class definitions
+- `/resources` mustache template files for xml responses
+- `/routes` API endpoint definitions
```

Comment:
Do `/routes` subdirectories need to be added here, or is that a level of detail we can comfortably leave out?

Reply:
just did it for consistency, gone into more detail below, can delete if u think its pointless